rest_api/aggregation/database_side/stages.py

```diff
@@ -115,4 +115,12 @@ def merge_analysis_driver_procs(id_field, projection=None):
     for k in projection:
         group[k] = {'$first': '$' + k}
     stages.append({'$group': group})
+    stages.append(
+        lookup(
+            'analysis_driver_stages',
+            'most_recent_proc.proc_id',
+            'analysis_driver_proc',
+            'most_recent_proc.stages'
+        )
+    )
```

Comment:
We probably don't need it, come to think of it - the Dataset in Analysis-Driver doesn't use the aggregate endpoint. I guess ideally we'd be able to turn this on and off with the query string, but that's a whole other rabbit hole to jump down.